CHANGELOG.md

```diff
@@ -21,8 +21,6 @@
         ([#2376](https://github.com/Automattic/pocket-casts-android/pull/2376))
     *   Fix deselected chapters not synced correctly between different platforms
         ([#2357](https://github.com/Automattic/pocket-casts-android/pull/2357))
-    *   Adds download episode action button to shelf list in Now Playing
-        ([#2325](https://github.com/Automattic/pocket-casts-android/pull/2325))
```

Comment:
This shouldn't be removed from here. The change is still in `7.67`. Instead we should update `7.68` with the information about removing the feature.

Reply:
[7294bae](https://github.com/Automattic/pocket-casts-android/pull/2493/commits/7294bae97dd0631c7b6b0e692098989eeda548b4)